### PR TITLE
Fixing issue: [BUG] Update the Discord icon in the readme #1698

### LINKED ISSUE #1698: [BUG] Update the Discord icon in the readme
### Description

The Discord icon in the `icons.md` file is currently outdated. It should be fixed immediately.
You can get the Discord logo from here: https://discord.com/branding

### Screenshots

image

### Additional information

This is a good first issue. Please leave it open to beginners or first-time contributors. 🙂
Do let us know if you're having any problems or any questions.

### PR DESCRIPTION
**Pull request by Pathholder1806**
## Fixes Issue

Closes #1698

## Changes proposed

I have updated the discord icon in the 'icons.md' file
New Icon Link: https://assets-global.website-files.com/6257adef93867e50d84d30e2/625e5f775305fb9733389e0f_Discord-Logo-Black%20(1).png
Ref: https://discord.com/branding

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1699"></a>

### CODE REVIEW

File: icons.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
 |  Codewars   |  codewars   | <img src="https://user-images.githubusercontent.com/76241195/147094572-962f911f-9ebd-4615-b72f-ff076f4033e8.png"  width=65% height=30%>                      |     DEV.to     |     dev.to     | <img src="https://user-images.githubusercontent.com/76241195/147094431-b194a188-0a63-4ce3-89c9-1d7578405318.png" width=100% height=30%> |
-|   Discord   |   discord   | <img src="https://user-images.githubusercontent.com/65664185/138502295-d82d98cf-2b42-4926-ab68-e45b2cfe8605.png" width=65% height=30%>                       |     Dollar     |     dollar     | <img src="https://user-images.githubusercontent.com/76985777/145393429-c03c5c3e-0416-4d28-be21-8a01a0c3dff5.png" width=100% height=30%> |
+|   Discord   |   discord   | <img src="https://assets-global.website-files.com/6257adef93867e50d84d30e2/625e5f775305fb9733389e0f_Discord-Logo-Black%20(1).png" width=65% height=30%>                       |     Dollar     |     dollar     | <img src="https://user-images.githubusercontent.com/76985777/145393429-c03c5c3e-0416-4d28-be21-8a01a0c3dff5.png" width=100% height=30%> |
```
Comment (Panquesito7) [inline icons.md:6]: This is a more black version of the icon. 🙂
```suggestion
|   Discord   |   discord   | <img src="https://user-images.githubusercontent.com/51391473/187728370-34aac260-8fea-4d41-aaeb-06b1643ccd9a.png" width=50% height=30%>                       |     Dollar     |     dollar     | <img src="https://user-images.githubusercontent.com/76985777/145393429-c03c5c3e-0416-4d28-be21-8a01a0c3dff5.png" width=100% height=30%> |
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — APPROVED** [review]
Awesome! Thank you. 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Maintainable

**Comment by Pathholder1806** [comment]
Thank you @Panquesito7 for your input on my change! Valuable inputs are always great way to learn. 🔥
Hoping to get the PR merged now soon😀

**Comment by kumarsonsoff3** [comment]
Please resolve the merge conflicts with your branch, let us know if you need any help with this! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
The changes to the icon look good. Please separate your profile into a separate pull request as the changes are not related.

**Comment by Panquesito7** [comment]
> The changes to the icon look good. Please separate your profile into a separate pull request as the changes are not related.

It seems the CI automatically made those changes. You can see the commit co-author is GitHub Actions.
We should merge it because if we revert the changes, the CI will make the appropriate changes again.

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
These changes in profile are may be because the issues with prettier last day! 

I believe, you need to **Sync your fork**, this issue must be resolved then! ⚒

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by Pathholder1806** [comment]
I have synced my fork @kumarsonsoff3 !

**Comment by kumarsonsoff3** [comment]
Hey @Pathholder1806, Probably I was unclear! 🤦‍♂️ 
I mean with the branch from which the PR is coming, which is 9 commits behind!

image

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by Pathholder1806** [comment]
Hey @kumarsonsoff3 !
Thanks for clarifying. I have updated the **patch-1** branch too!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good, Thanks for contributing! 👍

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**